api/V1/common.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package celestia.node.v1.common;
+
+option go_package = "github.com/celestiaorg/celestia-node/api/v1/common";
+
+// Common status codes used across all services
+enum StatusCode {
+  // Reserve 0 for unspecified
+  STATUS_UNSPECIFIED = 0;
+  // Success codes (1-99)
+  STATUS_OK = 1;
+  // Error codes (100+)
+  // ERROR_DATA_NOT_AVAILABLE is returned if data availability sampling gurantees was required for request
+  // and failed. Keep in mind that data availability sampling will be ensured for all blocks in
+  // sampling window till the requested height
+  ERROR_DATA_NOT_AVAILABLE = 100;
```

Comment:
Is this only returned by the `SharesAvailable` method?

Reply:
Removed in https://github.com/celestiaorg/canonical-api/pull/2